neuralmonkey/nn/mlp.py

```diff
@@ -6,18 +7,19 @@ class MultilayerPerceptron(object):
     """ General implementation of the multilayer perceptron. """
 
     # pylint: disable=too-many-arguments
-    def __init__(self, mlp_input, layer_configuration, dropout_plc,
-                 output_size, name: str = 'multilayer_perceptron',
-                 activation_fn=tf.nn.relu) -> None:
+    def __init__(self, mlp_input: tf.Tensor, layer_configuration: List[int],
+                 dropout_keep_prob: float,
+                 output_size: int, train_mode: tf.Tensor,
+                 name: str = 'multilayer_perceptron',
+                 activation_fn=tf.nn.relu, ) -> None:
```

Comment:
a taky to hoď argument na řádku, jako je to na většině místech, který maj `too-many-arguments`...